index.html

```diff
@@ -157,11 +157,16 @@ <h2 id="meetings">Meetings</h2>
       <p>Upcoming meetings are visible at the 
       <a href="https://github.com/w3c/ttwg/projects/1">TTWG project dashboard</a>.</p>
       <h2 id="recent-activity">Recent Activity and list of repositories</h2>
-      <p>Recent activity can be found on the following repository links:</p>
+      <p>
+        All of the TTWG's repositories can be found at <a href="https://www.w3.org/PM/Groups/repositories.html?gid=34314">Timed Text Working Group - Repositories</a> page.
+        Recent activity on the repositories where work is active at the moment can be found on the following repository links:
+      </p>
       <h3>Group repositories</h3>
       <ul>
         <li><a href="https://github.com/w3c/ttwg/pulse">TTWG business</a></li>
         <li><a href="https://github.com/w3c/tt-reqs/pulse">TTWG future requirements</a></li>
+        <li><a href="https://github.com/w3c/charter-timed-text/pulse">TTWG draft charter</a></li>
+        <li><a href="https://github.com/w3c/timed-text-dashboard/pulse">TTWG Tracker</a></li>
```

Comment:
Please remove these - nobody is interested in these at the moment I think.

Reply:
Thanks @himorin , looks like you removed the tracker but not the draft charter?

Reply:
Ooh.. I mis-thought that "this" points the nearest line but not all... (considered additional nearby line is shown by limitation of comment tool...)